### PR TITLE
User avatar instead identicon

### PR DESCRIPTION
**Pull request by vmangelovv**

### CODE REVIEW

File: src/popup/router/components/SidebarMenu.vue [hunk 1]
```diff
@@ -95,10 +95,13 @@
...
 import Arrow from '../../../icons/arrow-current-color.svg?vue-component';
+import UserAvatar from './UserAvatar'
 
 export default {
-  components: { Close, Arrow },
-  computed: mapGetters(['account', 'activeAccountName']),
+  components: { Close, Arrow, UserAvatar },
+  computed: {
+    ...mapGetters(['account', 'activeAccountName']),
+  },
```
Comment (davidyuk) [inline src/popup/router/components/SidebarMenu.vue]: This change is not necessary

File: src/popup/router/pages/Names.vue [hunk 2]
```diff
@@ -13,7 +13,8 @@
         <hr />
         <ae-list v-if="registeredNames.length">
           <ae-list-item fill="neutral" v-for="(name, key) in registeredNames" :key="key">
-            <ae-identicon v-bind:address="name.owner" size="base" />
+            <!-- <ae-identicon v-bind:address="name.owner" size="base" /> -->
```
Comment (davidyuk) [inline src/popup/router/pages/Names.vue]:
```suggestion
```
Let's remove the outdated code here and later completely

File: src/popup/router/pages/Names.vue [hunk 3]
```diff
@@ -47,7 +48,8 @@
 
         <ae-list v-if="!moreAuInfo.visible && activeAuctions != null">
           <ae-list-item class="singleAuction" fill="neutral" v-for="(info, key) in auctions" :key="key" @click="moreAuctionInfo(key, info)">
-            <ae-identicon class="subAccountIcon" v-bind:address="info.winning_bidder" size="base" />
+            <!-- <ae-identicon class="subAccountIcon" v-bind:address="info.winning_bidder" size="base" /> -->
```
Comment (mradkov) [inline src/popup/router/pages/Names.vue]:
```suggestion
```

File: src/popup/router/pages/Names.vue [hunk 4]
```diff
@@ -66,7 +68,8 @@
             <hr />
             <span>{{ $t('pages.namingSystemPage.currentBid') }}</span>
             <ae-list-item style="border:none" fill="neutral">
-              <ae-identicon class="subAccountIcon" v-bind:address="moreAuInfo.info.winning_bidder" size="base" />
+              <!-- <ae-identicon class="subAccountIcon" v-bind:address="moreAuInfo.info.winning_bidder" size="base" /> -->
```
Comment (mradkov) [inline src/popup/router/pages/Names.vue]:
```suggestion
```

Clean commented code.

File: src/popup/router/pages/Popups/PopupConnect.vue [hunk 5]
```diff
@@ -11,15 +11,17 @@
         <ae-icon name="check" />
       </div>
       <div class="identicon">
-        <ae-identicon :address="account.publicKey" size="base" />
+        <!-- <ae-identicon :address="account.publicKey" size="base" /> -->
```
Comment (mradkov) [inline src/popup/router/pages/Popups/PopupConnect.vue]:
```suggestion
```

File: src/popup/router/pages/Popups/PopupConnect.vue [hunk 6]
```diff
@@ -11,15 +11,17 @@
...
+        <UserAvatar :address="account.publicKey" size="lg" />
         <div class="accountName">{{ activeAccountName }}</div>
       </div>
     </div>
 
     <h2>
       <span class="secondary-text" data-cy="aepp">{{ data.host }} ({{ data.name }}) </span>
       {{ $t('pages.connectConfirm.websiteRequestconnect') }}
-      <ae-identicon class="send-account-icon" :address="account.publicKey" size="s" />
+      <!-- <ae-identicon class="send-account-icon" :address="account.publicKey" size="s" /> -->
```
Comment (mradkov) [inline src/popup/router/pages/Popups/PopupConnect.vue]:
```suggestion
```

File: src/popup/router/pages/Popups/PopupSignTx.vue [hunk 7]
```diff
@@ -3,14 +3,16 @@
     <ae-list class="spendTxDetailsList">
       <ae-list-item fill="neutral" class="flex-justify-between noBorder">
         <div class="flex flex-align-center accountFrom">
-          <ae-identicon :address="account.publicKey" />
+          <!-- <ae-identicon :address="account.publicKey" /> -->
```
Comment (mradkov) [inline src/popup/router/pages/Popups/PopupSignTx.vue]:
```suggestion
```

File: src/popup/router/pages/Popups/PopupSignTx.vue [hunk 8]
```diff
@@ -3,14 +3,16 @@
...
+          <UserAvatar :address="account.publicKey" />
           <span class="spendAccountAddr">{{ activeAccountName }}</span>
         </div>
         <div class="arrowSeprator">
           <ae-icon name="left-more" />
         </div>
         <div class="flex flex-align-center accountTo" v-if="isAddressShow">
-          <ae-identicon :address="receiver" />
+          <!-- <ae-identicon :address="receiver" /> -->
```
Comment (mradkov) [inline src/popup/router/pages/Popups/PopupSignTx.vue]:
```suggestion
```

File: src/popup/router/components/SidebarMenu.vue (hunk 1 above)
Comment (mradkov) [reply src/popup/router/components/SidebarMenu.vue]: Agree, its readable enough just like it was on a single line.

**Comment by davidyuk** [comment]
I have rebased it on the top of master and fixed linter errors

**Pull request merged**